packages/curve-ui-kit/src/features/slippage-settings/ui/SlippageSettingsModal.tsx

```diff
@@ -128,9 +128,12 @@ export const SlippageSettingsModal = ({ isOpen, maxSlippage, onSave, onClose }:
    */
   const [lastError, setLastError] = useState<Error | undefined>(undefined)
 
+  // Update form values when slippage changes or modal is closed without saving and re-opened
```

Comment:
hmm usually this would be handled automatically by de-mounting the modal. But mui likes to always render stuff :thinking: